client/pages/Assessment/pages/Section/components/DataTable/Table/Row/RowData/Cell/Cell.tsx

```diff
@@ -28,12 +29,13 @@ type Props = {
   disabled: boolean
   rowIndex: number
   col: Col
+  row: Row
 }
 
 const Cell: React.FC<Props> = (props) => {
-  const { data, assessmentName, sectionName, table, disabled, rowIndex, col } = props
+  const { data, assessmentName, sectionName, table, disabled, rowIndex, col, row } = props
 
-  const datum = (data?.[rowIndex] as any)?.[col.props.index as number]
+  const datum = data[_camelCase(row.props.variableName ?? '')]?.[col.props.colName]?.raw
```

Comment:
Extracting the data

Reply:
create a utils objects to extract raw data. getRawData ({countryIso, tableName, variableName, colName})